ncl/ncl_entries/dewtemp.ipynb

```diff
@@ -0,0 +1,123 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dewtemp_trh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "NCL's [`dewtemp_trh`](https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml) calculates the dew point temperature given temperature and relative humidity using the equations from John Dutton's _\"Ceaseless Wind\"_ (pg. 273-274) and returns a temperature in Kelvin\n",
+    "\n",
+    ":::{important}\n",
+    "To convert from Kelvin to Celsius `-273.15` and to convert from Celsius to Kelvin `+273.15`\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grab and Go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "6.298141316024157"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Input: Single Value\n",
+    "from geocat.comp import dewtemp\n",
+    "\n",
+    "kelvin = 273.15  # Kelvin\n",
+    "temp_c = 18  # Celsius\n",
+    "relative_humidty = 46.5  # %\n",
+    "\n",
+    "dewtemp(temp_c + kelvin, relative_humidty) - kelvin  # Returns in Celsius"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([  6.29814132, -35.12955277,  86.22114845, -27.40981025])"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Input: List/Array\n",
+    "from geocat.comp import dewtemp\n",
+    "\n",
+    "temp_kelvin = [291.15, 274.14, 360.3, 314]  # Kelvin\n",
+    "relative_humidity = [46.5, 5, 96.5, 1]  # %\n",
+    "\n",
+    "dewtemp(temp_kelvin, relative_humidity) - 273.15  # Returns in Celsius"
```

Comment:
```suggestion
    "dewtemp(temp_kelvin, relative_humidity) - kelvin  # Returns in Celsius"
```

Minor change, but wondering if we want to be consistent with the other code cell here.